### PR TITLE
Removes Fedora 25 Dockerfile, Adds Fedora 27 Dockerfile

### PR DESCRIPTION
**Pull request by seemethere**
Fedora 25 was EOL on December 12, 2017

https://fedoramagazine.org/fedora-25-end-life/

Signed-off-by: Eli Uriegas <eli.uriegas@docker.com>

## PR Summary

Remove Fedora 25, Add Fedora 27

## PR Checklist

Note: Please mark anything not applicable to this PR `NA`.

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [x] Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] User facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - [ ] Issue filed - Issue link:
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
    - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

File: docker/release/fedora27/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Docker image file that describes an fedora 25 image with PowerShell installed from Microsoft YUM Repo
+# Docker image file that describes an fedora 27 image with PowerShell installed from Microsoft YUM Repo
 
-FROM fedora:25
+FROM fedora:27
 
 ARG POWERSHELL_VERSION=6.0.0
-ARG IMAGE_NAME=microsoft/powershell:fedora25
+ARG IMAGE_NAME=microsoft/powershell:fedora26
```
Comment (TravisEz13) [inline docker/release/fedora27/Dockerfile]: If its fedora 27, why are we calling it `fedora26`?

**Comment by TravisEz13** [comment]
If this is officially supported, it should be added here https://github.com/PowerShell/PowerShell/blob/master/docker/tests/containerTestCommon.psm1#L68 as well.

**Comment by TravisEz13** [comment]
Based on this list it is.  https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md#linux

Please also file an issue to update the build and mention me and this PR.

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
Address comment about the tag in arg and about updating tests.

File: docker/release/fedora27/Dockerfile (hunk 1 above)
Comment (seemethere) [reply docker/release/fedora27/Dockerfile]: Changed

**Comment by TravisEz13** [comment]
filed #5989 for unrelated docs changes that are needed.

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
I'll leave this non-blocking PR open for review for at least 24 hours per governance.

**Comment by TravisEz13** [comment]
I reset your branch to before you added the docker tests because we need to build the images first.

**Comment by TravisEz13** [comment]
Thanks for your contribution!

**Pull request merged**